code/datums/diseases/advance/symptoms/atmospherics.dm

```diff
@@ -0,0 +1,80 @@
+/*
+//////////////////////////////////////
+Asthmothia
+	Very very Noticable.
+	Decreases resistance.
+	Decreases stage speed.
+	Reduced transmittability.
+BONUS
+	Spawns gases
+//////////////////////////////////////
+*/
+/datum/symptom/asthmothia
+	name = "Asthmothia"
+	stealth = -3
+	resistance = -4
+	stage_speed = -4
+	transmittable = -4
+	level = 8
+	severity = 6
+
+/datum/symptom/asthmothia/Activate(var/datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/turf/T = get_turf(A.affected_mob)
+		switch(A.stage)
+			if(2)
+				A.affected_mob << "<span class='warning'>You try to cough out excess gases.</span>"
```

Comment:
This seems a little obvious for a stage 2 message. Maybe have it be you start violently coughing or something.